Lib/ufo2ft/featureWriters/baseFeatureWriter.py

```diff
@@ -119,6 +153,110 @@ def _write(self):
         """Subclasses must override this."""
         raise NotImplementedError
 
+    def _insert(
+        self,
+        feaFile,
+        classDefs=None,
+        anchorDefs=None,
+        markClassDefs=None,
+        lookups=None,
+        features=None,
+    ):
+        """
+        Insert feature, its classDefs or markClassDefs and lookups at insert
+        marker comment.
+
+        If the insert marker is at the top of a feature block, the feature is
+        inserted before that block, and after if the insert marker is at the
+        bottom.
+        If the insert marker is in the middle of a feature block, that block
+        is split in to two blocks separated by the feature.
+        """
+
+        statements = feaFile.statements
+
+        # Collect insert markers in blocks
+        insertComments = self.context.insertComments
+
+        wroteOthers = False
+        for _, feature in features:
```

Comment:
if you're discarding the key of `features` all the time why passing in the items(), you seem to be only interested in the values